CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.7.1
+
+### Vision
+- Fixed dynamic `FPS` issue for a `Merge model`
+- Fix problem with neural networks run (swap `RGB channels`)
```

Comment:
I'd rephrase, 
```
- Fixed ModelPerformanceMode.DYNAMIC not taking effect with ModelPerformanceConfig.Merged
- Fixed detections of sign objects
```
so that it's clear to end user

Reply:
done